pkg/virt-controller/services/template.go

```diff
@@ -1370,3 +1411,17 @@ func readinessGates() []k8sv1.PodReadinessGate {
 		},
 	}
 }
+
+func CalculateBigUniqueValueForVmi(vmi *v1.VirtualMachineInstance) uint {
```

Comment:
I am not sure if this is guaranteed to be unique... also that the interval is "big enough" (we need to define "big" in this context). I think if you want to filter the process by some value which is unique for each VMI, then you can use the VMIs UID and pass it as an argument to some custom binary, which will then fall into an **infinite** sleep.

or ...

> One alternative to this workaround is to create another EmptyDir volume for the dedicated-cpu container and create a file with the right cgroup path / PID in it.

This alternative seems better to me if compared to calculation of unique sleep interval.

Reply:
:+1: 
If there are no objections, I would implement the emptyDir approach